### DISCUSSION TITLE
Unable to configure e-mail correctly

### DISCUSSION THREAD

**Discussion by arretx** [requarks/wiki#3443] [Help / Questions]
**Question**
I've been trying for 3 days to get the e-mail configuration to work and I keep getting either errors, or zero response from the test sending button, and the option to send e-mail when creating a new user is greyed out.

When I test send mail from the CLI in ubuntu using Postfix, it works without fail.  Not sure what to do at this point...but I need to be able to deploy documents to users, thus, I need to be able to add users and force password changes.  Ideas?

**Host Info (please complete the following information):**
OS: Ubuntu 20.04
Wiki.js version: 2.5.144
Database engine: Postgresql 12.4

**Comment by cyf-maple** [comment]
Did you solve this problem? Why does nothing happen when I click the test sending button, no email is received, and no error log.
